### PR TITLE
Adds an FAQ section

### PR DESCRIPTION
**Pull request by Monkiey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,26 @@ Note: Apple does not always release InstallAssistant.pkgs for the beta tracks an
 
 ### When will v1.0.0 Release?
 
-I'm glad you asked. This Saturday! If you want to get it early, you can join the [Ursinia Projects Discord Server](https://discord.gg/2DxVn4HDX6) and check `#announcements` where there will be a download link. If you find any bugs, report them there, not anywhere else.
+Its out check the releases tab
```
Comment (GrpeApple) [inline README.md:94]: `[Here](https://github.com/BenSova/Patched-Sur/releases/tag/v1.0.0).`
Also, please preserve history by doing `~~I'm glad you asked. This Saturday! If you want to get it early, you can join the [Ursinia Projects Discord Server](https://discord.gg/2DxVn4HDX6) and check `#announcements` where there will be a download link. If you find any bugs, report them there, not anywhere else.~~`.
Thought I might suggest that.

File: README.md [hunk 2]
```diff
@@ -91,7 +91,26 @@ Note: Apple does not always release InstallAssistant.pkgs for the beta tracks an
...
+
+### FAQ
+How to fix a prohibited symbol after I rest my PRAM?
+With V.1.0.0 of Pacthed Sur this issue is resloved once you patch the Bootplist
```
Comment (GrpeApple) [inline README.md:98]: Might have to remain consistent, and as the above, you can ignore this.

**Comment by GrpeApple** [comment]
Sounds a bit good, sounds a bit bad; cool anyways.

**Comment by GrpeApple** [comment]
Also I think that this should be in https://bensova.gitbook.io/big-sur/
But cool.

**Comment by BenSova** [comment]
Closing for now since I don't really want to have an FAQ with one section

**Pull request closed without merging**